app/javascript/packs/components/restaurants/deliveryTimeSelect.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import { withStyles } from '@material-ui/core/styles';
+
+import { formatDeliveryTime } from '../../modules/formatters/restaurants';
+
+const EMPTY_VAL = '';
+const DELIVERY_TIME = [
+    30 * 60 * 1000,
+    60 * 60 * 1000,
+    90 * 60 * 1000,
+    120 * 60 * 1000,
+];
```

Comment:
When working with times, I really recommend adding a suffix describing the unit, i.e: `DELIVERY_TIME_MS` or `_MILLIS` (or `_MIN`, or `_SEC`, etc.)

Reply:
Unless you have a time Duration class to unify all of that for you.